ndms2_client/client.py

```diff
@@ -134,7 +134,9 @@ def _parse_dict_lines(lines: List[str]) -> Dict[str, any]:
     for line in fixed_lines:
         # exploding the line
         colon_pos = line.index(':')
-        comma_pos = line.index(',') if ',' in line else None
+        comma_pos = None
+        if (ignoreComma == False):
```

Comment:
I don't get this approach. It's breaking the concept. 
If you expect some value in the dict to have comma, then we should limit the place we search for commas in using `end` argument for `index`